src/shared/components/palette/palette.module.ts

```diff
@@ -17,6 +17,7 @@ import { FormValidatorsService } from '../../services/form/form-validators.servi
 import { MarkdownModule } from '../markdown/markdown.module';
 import { AddressModule } from './address/address.module';
 import { BaseFieldModule } from './base-field/base-field.module';
+import { CaseFileViewModule } from './case-file-view';
```

Comment:
@johnbenjamin-hmcts - probably better to use the fully qualified form here, i.e. `'./case-file-view/case-file-view.module'`

Reply:
thank you @Daniel-Lam, I have now done the changes.